lib/modules/datasource/pod/readme.md

```diff
@@ -0,0 +1,38 @@
+This datasource will return releases from the Cocoapods public CDN by default.
+
+### Private Cocoapods Repositories
+
+It can also be configured to return releases from a private Cocoapods repository via the environment variable `COCOAPODS_GIT_REPOSITORIES` which takes a comma separated list of one or more repository URLs.
+These URLs should be exactly the same as what is in the Podfile of your project like so:
```

Comment:
Is this a proprietary environment variable invented to solve this requirement in Renovate? Or is it a standardized variable supported by Cocoapods itself?